css/team.css

```diff
@@ -60,7 +64,7 @@ Section Styles (background, padding, etc.)
 .wrapper {
 	width: var(--mobile-wrapper-width);
 }
-h5 {
+.h5-styles {
```

Comment:
Nice job adjusting the global h-styles! this is exactly how page css should interact with global styles!